drake/systems/plants/KinematicsCache.h

```diff
@@ -47,7 +47,9 @@ class KinematicsCacheElement
   }
 
 public:
-	EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+#ifndef SWIG
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
No (although I was originally concerned about that too). The SWIG symbol is just defined for the Swig preprocessor which scans the header files to figure out what to wrap. The C++ interface code that Swig creates still links against the original Drake libraries with the correct eigen allocators. For some reason, Eigen's macros were just confusing the preprocessor.